frontends/packages/livnote/desktop/components/notes/rich-text-editor.css

```
@@ -285,4 +285,146 @@
+  border-bottom: none;
+  background: transparent;
+}
+
+/* Comment highlighting animation for reverse selection */
+.comment-text-highlight {
+    animation: commentTextBlink 3s ease-in-out;
+}
+
+@keyframes commentTextBlink {
+    0% { 
+        background: rgba(255, 215, 0, 0.8) !important;
```

Biome (1.9.4)

[error] Using !important within keyframes declaration is completely ignored in some browsers.

Consider removing useless !important declaration.

(lint/suspicious/noImportantInKeyframe)